### PR TITLE
Create tesis-dan-disertasi-universitas-negeri-yogyakarta-pendidikan-p…

### PR DESCRIPTION
**Pull request by ahmadbahauddin**
…ascasarjana.csl

CSL untuk tesis dan disertasi PPs UNY 2017

### CODE REVIEW

**Comment by rmzelle** [comment]
@ahmadbahauddin, thanks. Is the style guide for this available online? We always try to add a "documentation" link (see https://github.com/citation-style-language/styles/wiki/Style-Requirements#10---documentation-link).

**Comment by ahmadbahauddin** [comment]
@rmzelle, your welcome. Yes, the documentation exists. sorry i'm forgot to included the documentation link. the pull request has been edited with documentation. 
documentation: https://drive.google.com/open?id=0B1fB-hcWFL4_SHNETkdrSmgwSGM

**Comment by ahmadbahauddin** [comment]
@rmzelle sorry the documentation is written in Indonesian.

File: tesis-dan-disertasi-universitas-negeri-yogyakarta-pendidikan-pascasarjana.csl [hunk 1]
```diff
@@ -0,0 +1,623 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never" default-locale="id-ID">
+  <!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Tesis dan Disertasi Universitas Negeri Yogyakarta Pendidikan Pascasarjana</title>
+    <title-short>TD-UNY-PPS</title-short>
+    <id>http://www.zotero.org/styles/tesis-dan-disertasi-universitas-negeri-yogyakarta-pendidikan-pascasarjana</id>
+    <link href="http://www.zotero.org/styles/tesis-dan-disertasi-universitas-negeri-yogyakarta-pendidikan-pascasarjana" rel="self"/>
+    <link href="http://www.zotero.org/styles/apa" rel="template"/>
+    <link href="https://drive.google.com/open?id=0B1fB-hcWFL4_SHNETkdrSmgwSGM" rel="documentation"/>
```
Comment (ahmadbahauddin) [inline tesis-dan-disertasi-universitas-negeri-yogyakarta-pendidikan-pascasarjana.csl]: documentation has been added.

**Comment by rmzelle** [comment]
> Yes, the documentation exists. sorry i'm forgot to included the documentation link. the pull request has been edited with documentation.
> documentation: https://drive.google.com/open?id=0B1fB-hcWFL4_SHNETkdrSmgwSGM

Do you know if it's okay to store a copy here on GitHub? I try to avoid linking to people's private online storage. (you can add an attachment to GitHub comments; see https://help.github.com/articles/file-attachments-on-issues-and-pull-requests/)

File: tesis-dan-disertasi-universitas-negeri-yogyakarta-pendidikan-pascasarjana.csl (hunk 1 above)
Comment (ahmadbahauddin) [reply tesis-dan-disertasi-universitas-negeri-yogyakarta-pendidikan-pascasarjana.csl]: [Pedoman Tesis dan Disertasi 2017_lengkap.pdf](https://github.com/citation-style-language/styles/files/1385096/Pedoman.Tesis.dan.Disertasi.2017_lengkap.pdf)
@rmzelle do yo mean like this?

**Comment by ahmadbahauddin** [comment]
@rmzelle wow, thanks. I don't know if I can store pdf on GitHub. the documentation has been added.

**Comment by ahmadbahauddin** [comment]
[Pedoman Tesis dan Disertasi 2017 revisi.pdf](https://github.com/citation-style-language/styles/files/1385103/Pedoman.Tesis.dan.Disertasi.2017.revisi.pdf)
@rmzelle, here is the documentation. the pull request has been edited. thanks.

**Comment by rmzelle** [comment]
@ahmadbahauddin, let me know if you're okay with the current style title. We always put the language in the title for non-English styles, and start the title with the university name.

The PDF and http://pps.uny.ac.id/ both use "Program Pascasarjana" instead of "Pendidikan Pascasarjana".

**Comment by rmzelle** [comment]
(otherwise this is good to go)

**Comment by ahmadbahauddin** [comment]
@rmzelle, I think you’re totally right about that. I'm okay with that one.

**Comment by rmzelle** [comment]
Great. Thanks for submitting!

**Comment by ahmadbahauddin** [comment]
@rmzelle, your welcome!

**Pull request merged**